src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -614,6 +615,18 @@ impl<'a> Linker for GccLinker<'a> {
             }
         }
     }
+
+    // Add the `GNU_EH_FRAME` program header which is required to locate unwinding information.
+    // Some versions of `gcc` add it implicitly, some (e.g. `musl-gcc`) don't,
+    // so we just always add it.
+    fn add_eh_frame_header(&mut self) {
+        // The condition here is "uses ELF" basically.
```

Comment:
What about mach-o?

Reply:
Ah, right, always forget that macos & friends exist.
That would be `self.sess.target.target.options.is_like_osx`.

Reply:
What's the resolution here? Mention macos in the comment, or filter `is_like_osx` from the `if`-condition too?

Reply:
Filter `is_like_osx` (I need to update the PR).

Reply:
Updated.